pycode/memilio-surrogatemodel/memilio/surrogatemodel/ode_secir_simple/model.py

```diff
@@ -0,0 +1,190 @@
+#############################################################################
+# Copyright (C) 2020-2022 German Aerospace Center (DLR-SC)
+#
+# Authors: Agatha Schmidt, Henrik Zunker
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+import numpy as np
+import pandas as pd
+import os
+import pickle
+import tensorflow as tf
+import matplotlib.pyplot as plt
+from memilio.surrogatemodel.ode_secir_simple import data_generation
+from memilio.surrogatemodel.ode_secir_simple import different_networks
+
+
+def plotCol(
+        inputs, labels, model=None, plot_col='InfectedSymptoms',
+        max_subplots=8):
+    """! Plot prediction of the model and label for one compartment.
+
+    @param inputs test inputs for model prediction. 
+    @param labels test labels. 
+    @param model trained model. 
+    @param plot_col string name of compartment to be plotted. 
+    @param max_subplots number of plot to be plotted. 
```

Comment:
This actually does not say anything. You probably mean "Number of the simulation run to be plotted and compared against"?